scripts/abi_artifact_generator/templates/setup_arguments.S.jinja_template

```diff
@@ -0,0 +1,53 @@
+/*#-
+This template file is distributed under the MIT License. See LICENSE.md for details.
```

Comment:
Make sure each `$MY_NAME.jinja_template` file generates a `$MY_NAME` file.
Currently this file generates `{msvc,gcc}_setup_arguments.S`.
Use directories to further organize the files.

Reply:
The idea was `${STYLE_NAME_IF_ANY}_${MY_NAME}`, but I guess directories work too.

Reply:
Yeah, but with so many templates, predictability makes life easier.